zerver/tests/test_email_mirror.py

```diff
@@ -735,3 +735,35 @@ def test_strip_from_subject(self) -> None:
         for subject in subject_list:
             stripped = strip_from_subject(subject['original_subject'])
             self.assertEqual(stripped, subject['stripped_subject'])
+
```

Comment:
`` TestStreamEmailMessageSubjectStripping `` probably isn't a good place for these tests, as it's focused on just testing stripping "RE"/"FWD"  combinations from the subject.